TanukiKit.xcodeproj/project.pbxproj

```diff
@@ -11,9 +11,9 @@
 		233A0B0E1C0EF8E900072A58 /* TanukiKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 233A0B031C0EF8E900072A58 /* TanukiKit.framework */; };
 		233A0B211C0EF94400072A58 /* Configuration.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1D1C0EF94400072A58 /* Configuration.swift */; };
 		233A0B221C0EF94400072A58 /* TanukiKit.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1E1C0EF94400072A58 /* TanukiKit.swift */; };
-		233A0B231C0EF94400072A58 /* Repositories.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1F1C0EF94400072A58 /* Repositories.swift */; };
+		233A0B231C0EF94400072A58 /* Projects.swift in Sources */ = {isa = PBXBuildFile; fileRef = 233A0B1F1C0EF94400072A58 /* Projects.swift */; };
```

Comment:
I know it bugs you that I named it `Repositories` instead of `Projects`. I agree with the change, but this makes this Pull Request almost impossible to review as the changes are not clear to me. Please rename this to `Repositories` again and we do the renaming in a separate Pull Request.